src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,20 @@
+import {Link} from "react-router-dom";
+
+
+function ArticleCard({article_id, title, topic, author, body, created_at, votes, comment_count}) {
+    return (
+        <Link to={`/articles/${article_id}`}>
+      <article className="article-card">
+        <h1>{title}</h1>
+        <h1>{topic}</h1>
+        <h1>{author}</h1>
+        <h1>{body}</h1>
+        <h1>{votes}</h1>
+        <h1>{comment_count}</h1>
```

Comment:
These should definitely not all be `h1` tags - there should only be one `h1` on a page and should probably be your header.
Consider using descending h values for the tags here as well as some simple `p` tags if possible